addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -50,7 +50,6 @@ patch(PosStore.prototype, {
     },
     async actionAfterIdle() {
         if (document.querySelector(".modal-open")) {
-            this.closeTempScreen();
```

Comment:
add something like `showscreen('floorscreen')`

Reply:
I think you need to do this.